test/integration/targets/wait_for/tasks/main.yml

```diff
@@ -165,9 +165,9 @@
       - waitfor is not changed
       - "waitfor.port == {{ http_port }}"
 
-- name: install psutil using pip (non-Linux only)
+- name: install psutil using pip (non-Linux and older FreeBSD only)
   pip:
-    name: psutil==5.8.0
+    name: psutil
```

Comment:
If the latest version (5.9.8) works, then just pin that version instead of using latest -- and then revert the change to the task name, which would no longer apply.